tests/testthat/test-ppc-distributions.R

```diff
@@ -286,3 +291,18 @@ test_that("ppc_violin_grouped renders correctly", {
 
   set.seed(seed = NULL)
 })
+
+test_that("ppc_pit_ecdf, ppc_pit_ecdf_grouped renders correctly", {
+  testthat::skip_on_cran()
+  testthat::skip_if_not_installed("vdiffr")
+
+  p_base <- ppc_pit_ecdf(y, yrep, interpolate_adj = FALSE)
+  g_base <- ppc_pit_ecdf_grouped(y, yrep, group = group, interpolate_adj = FALSE)
+  p_diff <- ppc_pit_ecdf(y, yrep, plot_diff = FALSE, interpolate_adj = FALSE)
+  g_diff <- ppc_pit_ecdf_grouped(y, yrep, plot_diff = FALSE, group = group, interpolate_adj = FALSE)
+
+  vdiffr::expect_doppelganger("ppc_pit_ecdf (default)", p_base)
+  vdiffr::expect_doppelganger("ppc_pit_ecdf_grouped (default)", g_base)
+  vdiffr::expect_doppelganger("ppc_pit_ecdf (no diff)", p_diff)
+  vdiffr::expect_doppelganger("ppc_pit_ecdf_grouped (no diff)", g_diff)
+})
```

Comment:
I see some files with no empty line at the end. Not a big deal but I would suggest to make it a habit to end a file with an empty line.